aiven/client/cli.py

```diff
@@ -67,6 +68,31 @@ def convert_str_to_value(schema, str_value):
         raise argx.UserError("Support for option value type(s) {!r} not implemented".format(schema["type"]))
 
 
+tag_key_re = re.compile(r"[\w\-]+")
+tag_value_re = re.compile(r"[\w\-,. ]*")
+
+
+def parse_tag_str(kv):
+    k, v = (kv.split(sep='=', maxsplit=1) + [''])[:2]
```

Comment:
Does the API treat the empty string `""` and `None` the same way? If not I suppose this should be `[None]` instead of `['']` since the user can define the empty string with `tag=`

Reply:
API expects always value, but it can be an empty string. `None` (omitting `"value"`) is not allowed. So both `tag=` and `tag` will produce empty string as tag value intentionally.